### ISSUE TITLE
MySQL 8 Initialize Error: BLOB/TEXT column 'identifier' used in key specification without a key length

### ISSUE THREAD

**Issue by hmmhmmhm** [authorizerdev/authorizer#156]
**Version:** 0.18.0

**Describe the bug**

```bash
2022/04/02 09:40:41 => version: 0.18.0
2022/04/02 09:40:41 using OS env instead of .env file
2022/04/02 09:40:41 Error while initializing db: Error 1170: BLOB/TEXT column 'identifier' used in key specification without a key length

```

**Steps To Reproduce**

1. Install MySQL 8 in Docker
2. Set-up the environment variable
3. Trying to initialize authorizer
4. Boom

**Expected behavior**

A normally initialized authorizer on mysql:8

**Comment by hmmhmmhm** [comment]
initialization was successful with postgresql. only mysql has been error.

**Comment by lakhansamani** [comment]
@hmmhmmhm thank you for reporting it.
Fixing it now

**Comment by lakhansamani** [comment]
@hmmhmmhm this is resolved in 0.18.1
